dvc/command/metrics.py

```diff
@@ -107,7 +107,11 @@ def add_parser(subparsers, parent_parser):
         "path", nargs="?", help="Path to a metric file or a directory."
     )
     metrics_show_parser.add_argument(
-        "-t", "--type", help="Type of metrics (raw/json/tsv/htsv/csv/hcsv)."
+        "-t",
+        "--type",
+        help="Type of metrics (json/tsv/htsv/csv/hcsv). \
```

Comment:
Let's format it something like:

```
help=(
    "Type of metrics (json/tsv/htsv/csv/hcsv)."
    "It can be detected by the file extension automatically."
    "Unsupported types will be treated as raw."
)
```